src/main/java/org/broadinstitute/hellbender/tools/walkers/featuremapping/AddFlowSNVQuality.java

```diff
@@ -530,20 +527,6 @@ final class SliceInfo {
         return ep;
     }
 
-    private double getSnvq(final double sliceP, final double p1, final double p2) {
-        if ( aqArgs.snvMode == AddFlowSNVQualityArgumentCollection.SnvqModeEnum.Legacy ) {
```

Comment:
@kenissur - please comment. We do not want to keep the strategies?

Reply:
I think we do want to keep the option to use the "legacy mode" where the hmers are assumed to be independent, what was the motivation to remove it?

Reply:
Mode restored.